src/components/maps/CameraViewer.jsx

```diff
@@ -0,0 +1,188 @@
+    this.renderMetadata = this.renderMetadata.bind(this);
+    this.renderDataPaging = this.renderDataPaging.bind(this);
+    this.changeDataPaging = this.changeDataPaging.bind(this);
+  
+    this.state = {
+      page: 0,
+    };
+  }
+
+  componentDidMount() {
+    this.setState({
+      page: 0,
+    });
+  }
+  
+  render() {
+    if (!this.props.mapConfig) return null;
+    
+    return (
+      <Layer className="camera-viewer">
+        <Box className="content" align="center">
+          <Box className="camera-metadata">
+            {this.renderMetadata()}
+          </Box>
+          <Box className="camera-data">
+            {this.renderDataPaging()}
+            {this.renderData()}
+          </Box>
+          <Button label="Done" onClick={()=>{ Actions.mapexplorer.resetActiveCamera() }}></Button>
+        </Box>
+      </Layer>
+    );
+  }
+
+  renderMetadata() {
+    if (this.props.activeCameraMetadataStatus === MAPEXPLORER_CAMERA_STATUS.FETCHING) {
+      return <SpinningIcon />;
+    } else if (this.props.activeCameraMetadataStatus === MAPEXPLORER_CAMERA_STATUS.ERROR) {
+      return <Notification message='ERROR' status='critical' />
+    }
+    
+    if (this.props.activeCameraMetadataStatus === MAPEXPLORER_CAMERA_STATUS.SUCCESS &&
+        this.props.activeCameraMetadata) {
+
+      return (
+        <List>
+        {Object.keys(this.props.activeCameraMetadata).map(key => (
+          <ListItem
+            key={`camera-metadata-${key}`}
+            justify='between'
+            separator='horizontal'
+          >
+            <span>{key}</span>
+            <span>{this.props.activeCameraMetadata[key]}</span>
+          </ListItem>
+        ))}
+        </List>
+      );
+    }
+
+    return null;
+  }
+
+  renderData() {
+    if (this.props.activeCameraDataStatus === MAPEXPLORER_CAMERA_STATUS.FETCHING) {
+      return <SpinningIcon />;
+    } else if (this.props.activeCameraDataStatus === MAPEXPLORER_CAMERA_STATUS.ERROR) {
+      return <Notification message='ERROR' status='critical' />
+    }
+    
+    if (this.props.activeCameraDataStatus === MAPEXPLORER_CAMERA_STATUS.SUCCESS &&
+        this.props.activeCameraData) {
+      const data = this.props.activeCameraData;
+      const items = [];
+      for (let i = this.state.page * ITEMS_PER_PAGE;
+           i < data.length && i < (this.state.page + 1) * ITEMS_PER_PAGE;
+           i++) {
+        let location = data[i].location || '';
+        
+        //Thumbnails!
+        //----------------
+        if (/zooniverse\.org/i.test(location)) {
+          location =
+            `${THUMBNAIL_SERVER_URL}${THUMBNAIL_WIDTH}x${THUMBNAIL_HEIGHT}/` +
+            location.replace(/^(https?|ftp):\/\//, '');
+        }
+        //----------------
+        
+        
+        items.push(
+          <Tile className="camera-data-item" key={`camera-data-${i}`}>
+            <Anchor href={data[i].location}>
+              <img src={location} />
```

Comment:
This should have an alt property for accessibility. Maybe something like `alt="Image from X location"` where X is replaced with the location name or camera id? Whatever makes sense. If there are no descriptions that make sense, then an empty alt property, `alt=""`, can be used to let screen readers know to skip trying to describe the image. The default behavior without any alt specified is to just read out the file name, which almost always isn't helpful.

Reply:
Hot damn, I can't believe I overlooked this.

Actually, it's _theoretically_ possible that I can create a fairly descriptive alt text, for example: _"Photo from SOBERANIA NATIONAL PARK at DAYTIME containing approximately ONE JAGUAR, we guess"_

That would require me to review the SQL calls, the project config, and the way aggregations are recorded in the DB, so I'll need to mark that as a future feature. For now, I'll adjust it enough for a fairly simple description, which unfortunately, wouldn't be anywhere as useful for screen readers. ("A photo from camera G01" is accurate but doesn't provide any useful context.)